src/modules/internal_resistance/internal_resistance.cpp

```diff
@@ -0,0 +1,224 @@
+#include "internal_resistance.hpp"
```

Comment:
Can you add the copyright header? See just about any other file in the code base.